src/function.rs

```diff
@@ -372,6 +377,34 @@ fn path_exists(evaluator: &Evaluator, path: &str) -> Result<String, String> {
   )
 }
 
+fn pick(_evaluator: &Evaluator, count: &str, alphabet: Option<&str>) -> Result<String, String> {
+  let alphabet = alphabet.unwrap_or(DEFAULT_RANDOM_ALPHABET);
```

Comment:
I prefer not to create constants that are only used in one place. This can just be `let alphabet = alphabet.unwrap_or("0123456789abcdef");`